app/dashboard/static/js/app/view-tests-all.2018.9.js

```diff
@@ -28,251 +31,134 @@ require([
-                distinct: 'lab_name',
-                query: query
+            qStr = URI.buildQuery({
+                'kernel': result.kernel,
+                'plan': plan,
             });
-        }
-
-        if (response.length > 0) {
-            batchOps = [];
-            response.forEach(createBatchOp);
-
-            deferred = request.post(
-                '/_ajax/batch', JSON.stringify({batch: batchOps}));
-
-            $.when(deferred)
-                .fail(error.error, getLabsCountFail)
-                .done(getLabsCountDone);
-        }
-    }
-
-    function getGroupsCountFail() {
-        html.replaceByClassHTML('groups-count-badge', '&infin;');
-    }
-
-    function getGroupsCountDone(response) {
-        var results;
-
-        // Internally used to parse the results.
-        function _updateGroupsCount(result) {
-            var count;
-            var opId;
-
-            count = parseInt(result.result[0].count, 10);
-            opId = result.operation_id;
-            gGroupsCount[opId] = count;
-
-            updateOrStageCount(opId, count);
-        }
-
-        results = response.result;
-        if (results.length > 0) {
-            results.forEach(_updateGroupsCount);
-            if (!gDrawEventBound) {
-                gDrawEventBound = true;
-                gTestsTable.addDrawEvent(updateTestsTable);
-            }
-        } else {
-            html.replaceByClassTxt('groups-count-badge', '?');
-        }
-    }
-
-    function getGroupsCount(response) {
-        var batchOps;
-        var deferred;
 
-        function createBatchOp(value) {
-            var group = value.board;
-            var query = gBatchOpBase;
-
-            query += group;
+            /* Number of test case regressions */
             batchOps.push({
                 method: 'GET',
-                operation_id: 'groups-count-' + group,
+                operation_id: 'status-' + result._id.$oid,
                 resource: 'count',
-                document: 'test_group',
-                query: query
+                document: 'test_regression',
+                query: qStr,
             });
         }
 
-        if (response.length > 0) {
-            batchOps = [];
-            response.forEach(createBatchOp);
-
-            deferred = request.post(
-                '/_ajax/batch', JSON.stringify({batch: batchOps}));
+        batchOps = [];
+        results.forEach(createBatchOp);
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
 
-            $.when(deferred)
-                .fail(error.error, getGroupsCountFail)
-                .done(getGroupsCountDone);
-        }
+        $.when(deferred)
+            .fail(error.error, getBatchStatusFailed)
+            .done(getBatchStatusDone)
     }
 
-    function getTestsDone(response) {
+    function updateTestTable(response) {
         var columns;
+        function _renderPlan(data, type, object) {
```

Comment:
Using a `renderKernel()` method to create a link to test plan results seems wrong.  Also, this is actually the same link as the default row link so I think we can just drop it.

Reply:
updated.